runtime/doc/news.txt

```diff
@@ -281,6 +281,10 @@ The following changes to existing APIs or features add new behavior.
   Note that syntax highlighting of code examples requires a matching parser
   and may be affected by custom queries.
 
+
+• "Q" now repeats a macro for each line of visual selection.
+• "@" now repeats the indicated macro for each line of visual selection.
```

Comment:
Nit, but I think it would make more sense to refer to this as code commands:

```suggestion
• `Q` now repeats a macro for each line of visual selection.
• `@` now repeats the indicated macro for each line of visual selection.
```